doc/source/building/index.rst

```diff
@@ -0,0 +1,354 @@
+        handled by the package managers.
+
+      .. tab-item:: CentOS/RHEL
+
+        To install SciPy build requirements, you can do::
+
+          sudo yum install gcc-gfortran python3-devel openblas-devel lapack-devel pkgconfig
+
+        Alternatively, you can do::
+
+          sudo yum-builddep scipy
+
+        This command installs whatever is needed to build SciPy, with the
+        advantage that new dependencies or updates to required versions are
+        handled by the package managers.
+
+      .. tab-item:: Arch
+
+        To install SciPy build requirements, you can do::
+
+          sudo pacman -S gcc-fortran openblas pkgconf
+
+  .. tab-item:: macOS
+
+    Install Apple Developer Tools. An easy way to do this is to
+    `open a terminal window <https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line>`_,
+    enter the command::
+
+        xcode-select --install
+
+    and follow the prompts. Apple Developer Tools includes Git, the Clang C/C++
+    compilers, and other development utilities that may be required.
+
+    Do *not* use the macOS system Python. Instead, install Python
+    with `the python.org installer <https://www.python.org/downloads/>`__ or
+    with a package manager like Homebrew, MacPorts or Fink.
+
+    The other system dependencies you need are a Fortran compiler, BLAS and
+    LAPACK libraries, and pkg-config. They're easiest to install with
+    `Homebrew <https://brew.sh/>`__::
+
+        brew install gfortran openblas pkg-config
+
+    .. note::
+
+        As of SciPy >=1.2.0, we do not support compiling against the system
+        Accelerate library for BLAS and LAPACK. It does not support a sufficiently
+        recent LAPACK interface. This is planned to change in 2023, because macOS
+        13.3 introduced a major upgrade to Accelerate which resolved all known
+        issues.
+
+  .. tab-item:: Windows
+
+    A compatible set of C, C++ and Fortran compilers is needed to build SciPy.
+    This is tricker on Windows than on other platforms, because MSVC does not
+    support Fortran, and gfortran and MSVC can't be used together. You will
+    need one of these sets of compilers:
+
+    1. MSVC + Intel Fortran (``ifort``)
+    2. Intel compilers (``icc``, ``ifort``)
+    3. Mingw-w64 compilers (``gcc``, ``g++``, ``gfortran``)
+
+    Compared to macOS and Linux, building SciPy on Windows is more difficult,
+    largely due to the difficulty of setting up these compilers. It is not
+    possible to just call a one-liner on the command prompt as you would on
+    other platforms.
+
+    First, install Microsoft Visual Studio - the 2019 Community Edition or any
+    newer version will work (see the
+    `Visual Studio download site <https://visualstudio.microsoft.com/downloads/>`__). 
+    This is needed even if you use the MinGW-w64 or Intel compilers, in order
+    to ensure you have the Windows Universal C Runtime.
```

Comment:
Is the Windows Universal C Runtime selected by default? In either case, it might be good to call out whether this is the *only* component that is needed so that the installation can be minimized, since a Visual Studio installation with lots of options can get quite large.

Reply:
Yes, I believe it is - will add that.